solr/core/src/test/org/apache/solr/cloud/TestPullReplica.java

```diff
@@ -128,7 +128,7 @@ public void tearDown() throws Exception {
   @Repeat(iterations = 30)
   public void testCreateDelete() throws Exception {
     try {
-      switch (2 /*random().nextInt(3)*/) {
+      switch (random().nextInt(3)) {
```

Comment:
[+1] This line by Alex fixes a mistake I made here.  I commented it out while testing a PR a week or two back, and it accidentally got added to my PR and merged.

Good catch Alex!